module/autopif.sh

```diff
@@ -110,7 +110,9 @@ if [ "$DIR" = /data/adb/modules/playintegrityfix/autopif ]; then
   cp -fv $NEWNAME ..;
 fi;
 
-if [ -f /data/adb/modules/playintegrityfix/killgms.sh ]; then
-  item "Killing any running GMS DroidGuard process ...";
-  sh /data/adb/modules/playintegrityfix/killgms.sh 2>&1;
+if [ "$DIR" = /data/adb/modules/playintegrityfix ]; then
```

Comment:
$DIR will be set to /data/adb/modules/playintegrityfix/autopif at this point for the use case you're trying to cover I think:
https://github.com/osm0sis/PlayIntegrityFork/blob/main/module/autopif.sh#L99

Reply:
Yup, sure is.  Apologies, updating.

Debug:
- My DIR variable is: /data/adb/modules/playintegrityfix/autopif